libs/components/action-bars/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "homepage": "https://github.com/blackbaud/skyux#readme",
   "peerDependencies": {
+    "@angular/animations": "^12.2.16",
```

Comment:
A note on this - this is not a breaking change even though it looks like it may be. I have to add this because I now import from the library. However, there is already a peer on `@skyux/animations` that had a peer of `@angular/animations` - so it isn't a breaking change as `@angular/animations` was already required by a required library.